docker-images/backend/Dockerfile

```diff
@@ -24,7 +24,7 @@ RUN adduser \
     appuser
 
 # Copy the source code into the container.
-COPY manage.py Makefile pyproject.toml poetry.lock /app/
+COPY manage.py Makefile pyproject.toml poetry.lock .flake8 .prettierignore .prettierrc.js /app/
```

Comment:
what's the goal of including `flake8` and `prettier` config files into docker container?